packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -211,11 +211,11 @@ class SelectableRegion extends StatefulWidget {
   const SelectableRegion({
     super.key,
     this.contextMenuBuilder,
+    this.magnifierConfiguration = TextMagnifierConfiguration.disabled,
```

Comment:
Should this really default to disabled?

Reply:
This change existed before this PR, I only moved the parameter around. I think this follows the same pattern as `EditableText` in that it doesn't provide a default being in the `widgets` library, and `SelectionArea` does provide a default since it sits in the `material` library.